tools/ansible-lint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint all Ansible playbooks, requires ansible-lint to be installed:
+# https://ansible-lint.readthedocs.io/en/latest/installing.html
+
+set -eu
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Find all YAML files that make a reference to ansible_env
+exitCode=0
+# We do want to read words here
+# shellcheck disable=SC2013
+for file in $(grep --include=\*.{yml,yaml} -rlw "$SCRIPT_DIR/.." -e "ansible_env"); do
```

Comment:
Typically, `ansible-lint` knows which are ansible files and which are not.   And it is better to run `ansible-lint` on an entire project, instead of a per file basis.  

Typically, all dot files should go in the root of the repo, in this case `.ansible-lint` is in `./tools/` by default `ansible-lint` will look for a configuration file in the root of the directory from where it is ran from.  In this case because the script is being triggered from the `Makefile` that is the root of the project.  

`ansible-lint` will return the proper exit code automatically based on the result of the command.  You can run `ansible-lint` from the root of the project and then `echo $?` to see the code.  

I think we simplify this to not have to use a script, and simply just call `ansible-lint` from within the `Makefile`

Reply:
Will do but it was picking up the antora playbook when I ran it before